### ISSUE TITLE
OnSendAddins for on-premise

### ISSUE THREAD

**Issue by dcegress** [MicrosoftDocs/office-docs-powershell#3056]
Regarding this statement below, CU6 in 2017 introduced this for on-premise, but it still says unsupported.  Is this now supported?

-OnSendAddinsEnabled

This parameter is available or functional only in the cloud-based service.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 43ca7906-9f33-2991-9da9-a626e5e45c46
* Version Independent ID: 5f388b3c-fb47-f7a0-e648-07cb3e59cf72
* Content: [Set-OwaMailboxPolicy (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/client-access/set-owamailboxpolicy?view=exchange-ps#feedback)
* Content Source: [exchange/exchange-ps/exchange/client-access/Set-OwaMailboxPolicy.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/client-access/Set-OwaMailboxPolicy.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@dcegress I confirmed that OnSendAddinsEnabled is available in Exchange 2016 CU12 and Exchange 2019 CU1, so I changed the parameter description accordingly.

Be advised the following other parameters that are also marked as cloud only are available in E16CU12 and E19CU1. Whether or not they work as intended is another story:

- FreCardsEnabled

- GroupCreationEnabled

- ThirdPartyAttachmentsEnabled

Thanks for the feedback.
